minijinja/src/vm/mod.rs

```diff
@@ -279,6 +279,18 @@ impl<'env> Vm<'env> {
                 };
             }
 
+            #[allow(unused_macros)]
```

Comment:
While I think this is a nice idea, at least the raw case should be allowed to use the `ok!` macro as mentioned.  I would probably just not do the `ok!` macro here.  Particularly with the fixes to `?` this codebase will avoid the `ok!` and `some!` macros in the future anyways.